File: editorial/chap01/answer_B02.cpp
#include <iostream>
using namespace std;
 
int main() {
	// 入力
	int A, B;
	cin >> A >> B;
	
	// 答えを求める
	bool Answer = false;
	for (int i = A; i <= B; i++) {
		if (100 % i == 0) Answer = true;
	}
	
	// 出力
	if (Answer == true) cout << "Yes" << endl;
	else cout << "No" << endl;
	return 0;
}

File: editorial/chap01/python/answer_B02.py
# 入力
A, B = map(int, input().split())

# 答えを求める
Answer = False
for i in range(A, B + 1):
	if 100 % i == 0:
		Answer = True

# 出力
if Answer == True:
	print("Yes")
else:
	print("No")
